engine/installation/linux/oracle.md

```diff
@@ -122,18 +131,30 @@ Docker from the repository.
     ```bash
     $ sudo yum makecache fast
     ```
-
-    If this is the first time you have refreshed the package index since adding
-    the Docker repositories, you will be prompted to accept the GPG key, and
-    the key's fingerprint will be shown. Verify that the fingerprint matches
-    `58118E89F3A912897C070ADBF76221572C52609D` and if so, accept the key.
-
+   
 2.  Install the latest version of Docker, or go to the next step to install a
     specific version.
 
     ```bash
     $ sudo yum -y install docker-engine
     ```
+    
+    If this is the first time you are installing a package from
+    the Docker repositories, the GPG key and fingerprint will be shown and
+    automatically accepted.
+    
+    If you do not want to automatically accept the GPG key, remove the `-y` 
+    parameter so that yum prompts you to accept the GPG manually.
+    
+    Ensure the GPG details match the following:
+    
+    ```
```

Comment:
Please add a `none` hint here.  Otherwise, this could be interpreted as something off the wall, like YAML.